py3status/modules/xrandr_rotate.py

```diff
@@ -29,6 +29,13 @@
     {icon} a rotation icon, specified by `horizontal_icon` or `vertical_icon`.
 
 
+Remarks:
+    We have seen a case when rotating a screen using this module crashed i3 session.
```

Comment:
I'm not sure the `Experimental` tag is helpful, because it is not clear at which point we should remove this tag. I don't think we have any statistics on how many people successfully use any specific modules, do we?

In my mind, if we are to write something, it should be a message with very specific instructions, like "please look for *this* and *this*, and if you see *this*, do exactly *this* and *this*". Or don't write anything at all, let people create Github issues with their descriptions, and we will investigate case by case.